server/repository/src/mock/invoice_line.rs

```diff
@@ -174,6 +174,31 @@ pub fn mock_outbound_shipment_d_invoice_lines() -> Vec<InvoiceLineRow> {
     vec![mock_outbound_shipment_d_invoice_line_a]
 }
 
+pub fn mock_outbound_shipment_no_stock_invoice_lines() -> Vec<InvoiceLineRow> {
+    let mock_outbound_shipment_no_stock_invoice_line_a: InvoiceLineRow = InvoiceLineRow {
+        id: String::from("outbound_shipment_no_stock_line_a"),
+        invoice_id: String::from("outbound_shipment_no_stock"),
```

Comment:
i've been using id from associated invoice: `mock_outbound_shipment_no_stock().id`, up to you but I think it makes it easier to see where associated record is defined (and references from the associated record), similar with item_id